app/controllers/admin/content_controller.rb

```diff
@@ -27,6 +27,21 @@ def new
     new_or_edit
   end
 
+  def merge_with
+    @article = Article.new(params[:article])
+    @merged_article = Article.find(params[:merge_with])
```

Comment:
Yes, I am still working on the article merging. This is only to get my first tests passing @kariabancroft